Sources/NIO/SocketAddresses.swift

```diff
@@ -119,14 +119,14 @@ public enum SocketAddress: CustomStringConvertible {
     }
 
     /// Returns the protocol family as defined in `man 2 socket` of this `SocketAddress`.
-    public var protocolFamily: Int32 {
+    public var protocolFamily: NIOBSDSocket.ProtocolFamily {
```

Comment:
This type change is incompatible: we need to preserve the old API where possible as well as the new one. Feel free to deprecate-and-replace though.

Reply:
Yeah, that change is from the previous `ProtocolFamily` change.  This will disappear when rebased.